libs/designsystem/src/lib/components/modal/modal-wrapper/compact/modal-compact-wrapper.component.ts

```diff
@@ -23,8 +23,8 @@ export class ModalCompactWrapperComponent implements Modal, OnInit {
   private ionModalElement: HTMLIonModalElement;
   private readonly ionModalDidPresent = new Subject<void>();
   private readonly ionModalWillDismiss = new Subject<void>();
-  readonly didPresent = this.ionModalDidPresent.toPromise();
-  readonly willClose = this.ionModalWillDismiss.toPromise();
+  readonly didPresent = lastValueFrom(this.ionModalDidPresent);
+  readonly willClose = lastValueFrom(this.ionModalWillDismiss);
```

Comment:
Why lastValue here - modal-wrapper utilizes firstValue?

Reply:
Thank you, I have corrected this to use `firstValueFrom` here too!